arc/src/main.c

```diff
@@ -388,6 +388,250 @@ static void handle_glcd(struct zjs_ipm_message* msg)
     ipm_send_reply(msg);
 }
 
+#ifndef DEBUG_BUILD
```

Comment:
I think you meant #ifdef? This is used only in DEBUG_BUILD below AFAICT. Bad Jimmy, didn't build with DEBUG! :)